src/renderers/WebGLRenderer.js

```diff
@@ -193,6 +193,9 @@ class WebGLRenderer {
 		let _clippingEnabled = false;
 		let _localClippingEnabled = false;
 
+		// transmission render target scale
+		this.transmissionRenderTargetScale = 1.0;
```

Comment:
Do you mind documenting this new property here: https://threejs.org/docs/index.html#api/en/renderers/WebGLRenderer

Reply:
Done, let me know if the wording is clear enough.